gix/src/repository/config/transport.rs

```diff
@@ -405,6 +405,14 @@ impl crate::Repository {
                         }
                     }
 
+                    {
+                        let key = "http.sslVerify";
+                        opts.ssl_verify = config
+                            .boolean_filter_by_key(key, &mut trusted_only)
```

Comment:
Please take a look around and note that each of the keys above in some way references a respective static version of the key they access. This must be done here as well, and there should be a `Http::SSL_VERIFY` constant.

Reply:
I added a debug_assert to ensure the key match the constants, is that what you meant?

Reply:
That's close. The question is why it shouldn't (potentially) fail if the value can't be read. [In this line](https://github.com/Byron/gitoxide/pull/1135/commits/778e597100740651fefff18c38c6522b681e180b#diff-9fe748aff5bf996a7a4dd05505aca13d725e2e6766613b1ccfbc3e22679120d8R344) there is an example for how to do this.

Reply:
Okay I understand what you mean, I added it.